CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+Contributting to IntelliJ IDEA
+===
+
+Opening a PR, please make sure you've read these articles:
+- [IntelliJ Coding Guidelines](http://www.jetbrains.org/display/IJOS/IntelliJ+Coding+Guidelines)
+- [Contributor License Agreement](http://www.jetbrains.org/display/IJOS/Contributor+Agreement)
+- [Contribute Code](http://www.jetbrains.org/display/IJOS/Contribute#Contribute-ContributeCode)
+
+Additionally, you **NEED** to have a signed CLA sent to JetBrains before your
+code can be merged, You can download one from [Contributor License Agreement](http://www.jetbrains.org/display/IJOS/Contributor+Agreement)
+
+Building the IDE
+==
+
```

Comment:
I honestly have no clue what I should do with this feedback. Please be more detailed.